src/zope/configuration/xmlconfig.py

```diff
@@ -162,7 +162,11 @@ class ConfigurationHandler(ContentHandler):
 
     def __init__(self, context, testing=False):
         self.context = context
-        self.testing = testing
+        # We use this in an `except:` clause. For backwards
+        # compatibility, ever though this attribute isn't documented,
+        # we keep it in an attribute named 'testing' which can be interpreted
+        # as a boolean
+        self.testing = BaseException if testing else ()
```

Comment:
I don't like this.

Reply:
So what happens when somebody does `handler.testing = True`? ;)

Reply:
That's not tested :) There are tests for its boolean value though.